### DISCUSSION TITLE
Binaries

### DISCUSSION THREAD

**Discussion by allquixotic** [schibsted/jslt#186] [Ideas]
Would it be possible to release binaries for at least a few platforms? What I'd like to see is:

- Built shadow JAR of latest release
- GraalVM build (using official JDK11 latest GraalVM release) for Win64-MSVC
- GraalVM build for x86-64 CentOS/RHEL 7.x (should also work on a plethora of other distros, including 8.x and the last 2-3 Ubuntu LTS releases)
- GraalVM build for x86-64 MacOS current-1 (should also work on M1 via Rosetta, and on the latest MacOS release)

I was able to compile using GraalVM myself, but the environment setup takes a bit of work.

Prerequisites:

- Set the manifest in the shadow jar to reference the Main-Class

**Comment by larsga** [comment]
A shadow jar we could definitely build and release. (Once the CLI is moved out into a separate artifact etc.)

The GraalVM binaries look like they're perhaps a bit more hassle. I see there are Gradle plugins for building such binaries, so if someone were to contribute a PR to automate those builds we could do it.
